src/Jadu/Pulsar/Twig/Extension/GetConstantExtension.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Jadu\Pulsar\Twig\Extension;
+
+use Twig_Extension;
+use Twig_SimpleFunction;
+
+class GetConstantExtension extends Twig_Extension
+{
+    public function getFunctions()
+    {
+        return array(
+            new Twig_SimpleFunction('get_constant', array($this, 'getConstant')),
+        );
+    }
+
+    /**
+     * Name of this extension
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return 'get_constant_extension';
+    }
+
+    /**
+     * Get constant value if defined, null if not
+     *
+     * @param  string $name The name of the constant
+     * @return string       The value of the constant, null if not defined
+     */
+    public function getConstant($name = '')
```

Comment:
Might it be good to have a fallback value passed by parameter, instead of always returning null? e.g. if a string is wanted. Just add `, $fallback = null` to the args list and `return $fallback` on line 36